scripts/grunt/default_task.js

```diff
@@ -34,6 +34,7 @@ module.exports = function(grunt) {
   grunt.registerTask('no-only-tests', function() {
     var files = grunt.file.expand(
       'public/**/*@(_specs|.test).@(ts|js|tsx|jsx)',
+      'packages/grafana-data/**/*@(_specs|.test).@(ts|js|tsx|jsx)',
       'packages/grafana-ui/**/*@(_specs|.test).@(ts|js|tsx|jsx)',
       'packages/grafana-runtime/**/*@(_specs|.test).@(ts|js|tsx|jsx)'
```

Comment:
```suggestion
      'packages/**/*@(_specs|.test).@(ts|js|tsx|jsx)'
```

Reply:
I think we could get rid of package specific paths here in favour of assuring the same thing in the entire `packages` dir.